external/build/fugu-showcase.js

```diff
@@ -0,0 +1,176 @@
+/**
+ * @fileoverview Fetches the Fugu Projects following the data process described
+ * in https://github.com/GoogleChromeLabs/fugu-showcase#data-flow.
+ */
+
+require('dotenv').config();
+
+const {default: fetch} = require('node-fetch');
+const path = require('path');
+const {unfurl} = require('unfurl.js');
+const sharp = require('sharp');
+const fs = require('fs').promises;
+
+const SPREADSHEET_URL =
+  'https://sheets.googleapis.com/v4/spreadsheets/1S_Apr0HavFCO7H9hKcRjIUrgoT7MFRg4uBm7aWSoaYo/values/Sheet2?key=AIzaSyCkROWBarEOJ9hQJggyrlUFulOFA4h6AW0&alt=json';
+
+const SCREENSHOT_OPTIONS = {
+  width: 1280,
+  height: 800,
+  overwrite: true,
+  delay: 2,
+  type: 'webp',
+};
+
+async function limit(tasks, concurrency) {
+  const results = [];
+
+  async function runTasks(tasksIterator) {
+    for (const [index, task] of tasksIterator) {
+      try {
+        results[index] = await task();
+      } catch (error) {
+        // @ts-ignore
+        results[index] = new Error(`Failed with: ${error.message}`);
+      }
+    }
+  }
+
+  const workers = new Array(concurrency).fill(tasks.entries()).map(runTasks);
+  await Promise.allSettled(workers);
+  return results;
+}
+
+const createScreenshots = async (data, overrideType = null) => {
+  // eslint-disable-next-line node/no-unsupported-features/es-syntax
+  const captureWebsite = (await import('capture-website')).default;
+  let items = data.map(item => [item.appURL, item.screenshot]);
+  const length = items.length;
+  if (!overrideType) {
+    items = items.concat(items);
+  }
+  const tasks = items.map(([url, filename], i) => {
+    return () => {
+      SCREENSHOT_OPTIONS.type = overrideType || SCREENSHOT_OPTIONS.type;
+      SCREENSHOT_OPTIONS.darkMode = i >= length;
+      filename = SCREENSHOT_OPTIONS.darkMode
+        ? filename.replace(
+            new RegExp(`(.${SCREENSHOT_OPTIONS.type}$)`),
+            '-dark$1'
+          )
+        : filename;
+      if (!SCREENSHOT_OPTIONS.darkMode) {
+        data[i].screenshot = filename;
+      } else {
+        data[i - length].screenshotDark = filename;
+      }
+      // @ts-ignore
+      return captureWebsite.buffer(url, SCREENSHOT_OPTIONS).then(buffer => {
```

Comment:
Don't mix `.then` and `async/await`. As this method is `async` already, use `await` to make the code more readable and less nested.